modules/aws_ecs/variables.tf

```diff
@@ -202,6 +202,18 @@ variable "rds_instance_storage_encrypted" {
   description = "Whether the RDS instance should have storage encrypted. Defaults to false."
 }
 
+variable "rds_backup_retention_period" {
+  type        = number
+  default     = 0
+  description = "Sets RDS retention period for backups. Defaults to 0."
```

Comment:
maybe 30? by default?

Reply:
Probably overkill, but I was thinking of this from the perspective of merging this into the main repo we forked from. Defaulting to 0/null here retains the default behavior of the upstream behavior of no backups

Reply:
u right, good call